tests/test_commands.py

```diff
@@ -47,6 +47,17 @@ def run(self, conn):
 
 
 @pytest.fixture
+def mock_rethink_db_drop_not_exist(monkeypatch):
```

Comment:
Did you consider using `unittest.object.patch`? Might be simpler, you can see an example here:
 - https://github.com/bigchaindb/bigchaindb/blob/master/tests/pipelines/test_utils.py#L21